.github/workflows/test-c3.yml

```diff
@@ -78,3 +80,76 @@ jobs:
         env:
           CLOUDFLARE_API_TOKEN: ${{ secrets.C3_TEST_CLOUDFLARE_API_TOKEN }}
           CLOUDFLARE_ACCOUNT_ID: ${{ secrets.C3_TEST_CLOUDFLARE_ACCOUNT_ID }}
+
+  get-dependabot-bumped-framework:
+    runs-on: ubuntu-latest
+    outputs:
+      bumped-framework-cli: ${{ steps.detect.outputs.result }}
+    if: |
+      github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          ref: ${{ github.head_ref }}
+      - name: Get PR description as string
+        id: get-pr-description
+        run: |
+          str=$(sed 's/`/\`/g' <<EOF
+            ${{ github.event.pull_request.body }}
+          EOF
+          )
+          echo 'result<<EOF' >> $GITHUB_OUTPUT
+          echo $str >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
+      - name: detect-bumped-framework
+        id: detect
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const json = require('./packages/create-cloudflare/src/frameworks/package.json')
```

Comment:
If possible I think this would be better if it was a standalone script we invoked instead of inlined like it is. I think it would be easier to read and maintain that way.

Reply:
mh... the problem is that it would complicate the interaction with the workflow (passing the inputs to the js code and passing the outputs from it to the workflow)

I have an idea on how to do it in a hopefully clear way, I'll give that a shot 🙂👍

Reply:
Would a shell script make that any easier?

Either way, this isn't a huge concern and it's something we can worry about later. Curious to see your idea :D

Reply:
dang, I haven't even thought of a shell script 🙈 

but I guess it would have the same sort of issue as having javascript, meaning code written as a string (no syntax highlighting, etc...), so I am not sure if it would make a sizeable difference 🤷

Reply:
This is my idea: https://github.com/dario-piotrowicz/dependabot-testing/commit/5fc2d8554f14a63a20d1456791f6b17db05cc203

basically keeping all the best from the current implementation (simple inputs/outputs) but externalizing the js logic

what do you think could this work for you? 🙂

Reply:
PS: it seems like the action does allow the use of ESM but you need to specify `"type": "module"` in your package.json: https://github.com/actions/github-script#use-esm-import so I guess it's safer/simpler to just use cjs here